app/src/main/res/layout/onboarding_slide_final.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.onboarding.OnboardingSlideFinalViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+      android:id="@+id/slide_image_view"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      android:contentDescription="@string/onboarding_slide_3_title"
+      android:src="@drawable/ic_portrait_onboarding_3"
+      app:layout_constraintDimensionRatio="5:4"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+      android:id="@+id/slide_title_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="40dp"
+      android:layout_marginEnd="28dp"
+      android:layout_marginBottom="16dp"
+      android:fontFamily="sans-serif-medium"
+      android:gravity="center_horizontal"
+      android:maxLines="1"
+      android:text="@string/onboarding_slide_3_title"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/slide_image_view" />
+
+    <TextView
+      android:id="@+id/slide_description_text_view"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="48dp"
+      android:layout_marginTop="16dp"
+      android:layout_marginEnd="48dp"
+      android:fontFamily="sans-serif"
+      android:gravity="center_horizontal"
+      android:maxLines="2"
+      android:minLines="2"
+      android:text="@string/onboarding_slide_3_description"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/slide_title_text_view" />
+
+    <Button
+      android:id="@+id/get_started_button"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="56dp"
+      android:layout_marginEnd="56dp"
+      android:layout_marginTop="32dp"
```

Comment:
This is `36dp` in mocks.
 
<img width="436" alt="Screenshot 2020-04-28 at 11 41 23 AM" src="https://user-images.githubusercontent.com/9396084/80453109-4022e980-8945-11ea-9b00-cd97672d84e9.png">

Reply:
Done